app/javascript/components/admin/UserModal.js

```diff
@@ -131,6 +137,21 @@ class UserModal extends React.Component {
                 onChange={this.handleAPIAccessChange}
               />
             </Form.Group>
+            <Form.Group>
+              <Form.Label>Notes</Form.Label>
+              <InputGroup>
+                <Form.Control
+                  id="notes"
+                  as="textarea"
+                  rows="5"
+                  className="form-square"
+                  defaultValue={this.props.initialUserData.notes ? this.props.initialUserData.notes : ''}
```

Comment:
I think you can replace this line with `value={this.state.notes}` since this conditional is already in the constructor